server/utils/users.js

```diff
@@ -23,11 +24,17 @@ class Users {
   addUser(id, name, room, avatar) {
     const user = { id, name, room };
     user.avatar = this.saveAvatar(avatar);
+    user.color = this.assignColor(colors);
     this.users.push(user);
 
     return user;
   }
 
+  assignColor(colors) {
+    const randIdx = Math.floor(Math.random() * colors.length);
```

Comment:
I wonder if the Math.random() can be equal to 1 and what is the probability of this event. In theory, if it is possible, then this function can return undefined and the color of the title will be default one (in theory "black").